### DISCUSSION TITLE
base.blade.php

### DISCUSSION THREAD

**Discussion by carlituxman** [filamentphp/filament#591] [Help]
At base.blade.php

\Filament\Filament::getStyles()

return http instead https, the url is https

where/how create the path?

**Comment by ryangjchandler** [comment]
This is likely down to your `APP_URL` environment variable not using `https://`. If you want all URLs that are generated in your app to use `https`, you can add this to your `AppServiceProvider::boot()` method.

```php
public function boot()
{
    URL::forceScheme('https');
}
```
